hack/local-up-cluster.sh

```diff
@@ -450,6 +462,13 @@ function start_apiserver {
             AUTH_ARGS="--username=admin --password=admin"
         fi
     fi
+
+    # create the kube-public namespace for the aggregator
+    ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create namespace kube-public
```

Comment:
need a namespace to create the pod in.  In order to be publicly accessible, `kube-public` seems like a reasonable spot.